SharpPcap/LibPcap/PcapUnmanagedStructures.cs

```diff
@@ -219,18 +219,6 @@ public struct pcap_pkthdr_macosx
 
         #endregion
 
-        /// <summary>
```

Comment:
it could be helpful to put this change in its own commit to explain the reasoning here, it seems unrelated to removing the MAX_PACKET checks.

Reply:
It is related to MAX_PACKET_SIZE, see the attribute `[MarshalAs(UnmanagedType.ByValArray, SizeConst = SharpPcap.Pcap.MAX_PACKET_SIZE)]`

I should have changed the attribute, but the struct was not used so I just deleted it.

Reply:
ahh k. Can you put that in a separate commit if its unused? maybe remove it prior to adjusting the max_packet stuff, like the middle commit in the series?

Reply:
Done